### PR TITLE
Non-Res Exemptions Copy Updates

### PR DESCRIPTION
**Pull request by cameron-eyds**
*Issue #:* /bcgov/entity#21084

*Description of changes:*
- Update Exemptions and Non Res Exemptions Help Info Templates
- Update Non Res Exemptions Confirm Completion content

- Adds QoL function to set default flags to true for local development only - No longer required to do manually before and after completing new features.
- This requires a value in local .env in order to leverage, will operate as normal without .env value: `VUE_APP_LOCAL_DEV="true"`

Screenshot 2024-05-15 at 7 02 29 AM
Screenshot 2024-05-15 at 7 02 21 AM

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/utils/feature-flags.ts [hunk 1]
```diff
@@ -90,3 +90,15 @@ export async function updateLdUser (
 export function getFeatureFlag (name: string): any {
   return ldClient ? ldClient.variation(name, defaultFlagSet[name]) : defaultFlagSet[name]
 }
+
+/**
+ * Sets all boolean properties in the given default flag set object to true.
+ * @param defaultValue - The default flag value.
+ */
+export function setAllFlagDefaults(defaultValue: boolean): void {
```
Comment (dimak1) [inline ppr-ui/src/utils/feature-flags.ts:98]: Nice, what a timing, just this week I was thinking the same, how each time the flags needs to reset and what would be a good solution. This is neat! 👍🏻

**Review by dimak1 — APPROVED** [review]

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1873-4f3bqmgw.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1873-4f3bqmgw.web.app

File: ppr-ui/src/utils/feature-flags.ts (hunk 1 above)
Comment (cameron-eyds) [reply ppr-ui/src/utils/feature-flags.ts:98]: When you mentioned that earlier i thought the same thing, this was overdue!
Saves us remembering the defaults constantly 👍

Comment (dimak1) [reply ppr-ui/src/utils/feature-flags.ts:98]: I was even thinking of git ignoring the feature-flags file locally. https://stackoverflow.com/questions/49305201/gitignore-only-on-local

**Review by doug-lovett — APPROVED** [review]
Very nice.

**Pull request merged**